src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -222,14 +222,12 @@ private double ticksToFeet(double ticks) {
 		return feet;
 	}
 
-	public void setRamp(double elevatorHeight) {
-		double max = 4.0;
-		double min = 0.2;
-		double yintercept =Math.abs( ((357 * (max-min))/(RobotMap.ELEVATOR_BOTTOM_TICKS - RobotMap.ELEVATOR_TOP_TICKS)) - max);
-		double ramp = ((elevatorHeight * ((max-min) / (RobotMap.ELEVATOR_BOTTOM_TICKS - RobotMap.ELEVATOR_TOP_TICKS))))+ + yintercept;
+	public void setRamp(int elevatorHeight) {
+		double heightPercent = (double) (RobotMap.ELEVATOR_BOTTOM_TICKS - elevatorHeight) / (RobotMap.ELEVATOR_BOTTOM_TICKS - RobotMap.ELEVATOR_TOP_TICKS);
+		double ramp = MathUtils.weightedAverage(RobotMap.ELEVATOR_LOW_DRIVE_RAMP_TIME, RobotMap.ELEVATOR_HIGH_DRIVE_RAMP_TIME, heightPercent);
 
 		left.setOpenLoopRamp(ramp);
 		right.setOpenLoopRamp(ramp);
-		LOGGER.debug("TILT: "+ Gyrometer.getInstance().getPitchDegrees());
+		LOGGER.debug("Ramp: "+ ramp);
```

Comment:
Should be called "ramp time", since we have a Ramp class this log message could be confusing.